doc/examples/nonlinear_from_rest/submit_multiple_sim_sidewall.py

```diff
@@ -35,7 +37,7 @@
 if aspect_ratio in Ra_c_SW_tests.items():
     Ra_c_guessed = Ra_c_SW_tests[aspect_ratio]
 else:
-    Ra_c_guessed = 1.93e8 * aspect_ratio**-3.15
+    Ra_c_guessed = 1.93e8 * aspect_ratio ** -3.15
```

Comment:
Strange. Could you apply black on this PR and check that your black is up to date.

Reply:
I don't know. My black is up to date.